yb-voyager/src/utils/commonVariables.go

```diff
@@ -78,11 +78,12 @@ type SchemaReport struct {
 }
 
 type SchemaSummary struct {
-	DBName      string     `json:"DbName,omitempty"`
-	SchemaNames []string   `json:"SchemaNames,omitempty"`
-	DBVersion   string     `json:"DbVersion,omitempty"`
-	Notes       []string   `json:"Notes,omitempty"`
-	DBObjects   []DBObject `json:"DatabaseObjects"`
+	DBName        string     `json:"DbName,omitempty"`
+	SchemaNames   []string   `json:"SchemaNames,omitempty"`
+	DBVersion     string     `json:"DbVersion,omitempty"`
+	Notes         []string   `json:"Notes,omitempty"`
+	DBObjects     []DBObject `json:"DatabaseObjects"`
+	DBObjectsDesc string     `json:"DatabaseObjectsDesc,omitempty"`
```

Comment:
changing this struct will also reflect in the analyze schema report, right?

how about adding it in the `assessmentReport` struct as: `assessmentReportSchemaSummaryDesc`